.github/actions/hugo-build-versioned-action/action.yaml

```diff
@@ -12,17 +12,28 @@ inputs:
     required: false
     description: Environment variable
     default: production
+  keep_master:
+    required: false
+    description: Preview of the master branch
+    default: no
+  fetch-from:
+    required: false
+    description: repo to fetch the versions from
+    default: origin
 
 runs:
   using: "composite"
   steps:
     - name: Generate Versioned Documentation
-      shell: bash 
+      shell: bash
       run: |
         cd docs
-        ../scripts/generate.sh origin master
```

Comment:
This could've been injected, because we ran checkout in the action, that called this one